### PR TITLE
EDGOAIPMH-28 - Upgrade to edge-common v2.0.0

### PR DESCRIPTION
**Pull request by khandramai**
- edge-common changed to version 2.0.0
- unit tests updated and refactored

### CODE REVIEW

File: src/test/java/org/folio/edge/oaipmh/MainVerticleTest.java [hunk 1]
```diff
@@ -55,14 +56,12 @@
 
   private static final Logger logger = Logger.getLogger(MainVerticleTest.class);
 
-  private static final String apiKey = "Z1luMHVGdjNMZl9kaWt1X3VzZXI=";
-  private static final String apiKeyForTestUser = "Z1luMHVGdjNMZl90ZXN0X3Rlc3Q=";
-  private static final String badApiKey = "ZnMwMDAwMDAwMA==0000";
+  private static final String API_KEY = "eyJzIjoiajFFWkVaQlI3SSIsInQiOiJkaWt1IiwidSI6InVzZXIifQ==";
```
Comment (craigmcnally) [inline src/test/java/org/folio/edge/oaipmh/MainVerticleTest.java]: What I've done in other places is to generate the key on the fly, makes it clear what information is in the key:

`ApiKeyUtils.generateApiKey(10, "diku", "diku");`

File: src/test/java/org/folio/edge/oaipmh/MainVerticleTest.java [hunk 2]
```diff
@@ -55,14 +56,12 @@
...
+  private static final String API_KEY_FOR_TEST_USER = "Z1luMHVGdjNMZl90ZXN0X3Rlc3Q=";
```
Comment (craigmcnally) [inline src/test/java/org/folio/edge/oaipmh/MainVerticleTest.java]: pretty sure this would be considered a malformed API key

Comment (khandramai) [reply src/test/java/org/folio/edge/oaipmh/MainVerticleTest.java]: Thanks, Craig. Taken into account.

**Comment by khandramai** [comment]
[PR#100](https://github.com/folio-org/folio-api-tests/pull/100) contains corresponding api-tests changes.

**Pull request merged**